nuxeo-full-ps-template.yaml

```diff
@@ -0,0 +1,1989 @@
+        . /opt/nuxeo/connect/connect.properties
+        if [ -n "$NUXEO_CONNECT_USERNAME" -a -n "$NUXEO_CONNECT_PASSWORD" -a -n "$NUXEO_STUDIO_PROJECT" ]; then
+          echo "---> Registering instance on connect"
+          nuxeoctl register $NUXEO_CONNECT_USERNAME $NUXEO_STUDIO_PROJECT dev openshift $NUXEO_CONNECT_PASSWORD
+        fi
+      fi
+
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    annotations:
+      volume.beta.kubernetes.io/storage-class: shared
+    name: ${APPLICATION_NAME}-dev-binaries
+    labels:
+      app: ${APPLICATION_NAME}-dev
+      role: data
+      stage: dev
+  spec:
+    accessModes:
+    - ReadWriteMany
+    resources:
+      requests:
+        storage: ${VOLUME_BINARIES_CAPACITY}
+
+
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}-dev
+      stage: dev
+      component: nuxeo
+    name: ${APPLICATION_NAME}-dev-nuxeo
+  spec:
+    strategy:
+      type: Rolling
+      activeDeadlineSeconds: 21600
+      resources: {}
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+    replicas: 1
+    selector:
+      app: ${APPLICATION_NAME}-dev
+      component: nuxeo
+      stage: dev
+      deploymentconfig: ${APPLICATION_NAME}-dev-nuxeo
+
+    template:
+      metadata:
+        labels:
+          app: ${APPLICATION_NAME}-dev
+          component: nuxeo
+          stage: dev
+          deploymentconfig: ${APPLICATION_NAME}-dev-nuxeo
+      spec:
+        containers:
+        - image: ${APPLICATION_NAME}-nuxeo:latest
+          imagePullPolicy: Always
+          livenessProbe:
+            httpGet:
+              path: /nuxeo/runningstatus
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 30
+            timeoutSeconds: 5
+            periodSeconds: 10
+            successThreshold: 1
+            failureThreshold: 3
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /nuxeo/runningstatus
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 20
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 5
+          name: ${APPLICATION_NAME}-appserver
+          env:
+          - name: NUXEO_TEMPLATES
+            value: "default,mongodb,mongodb-audit"
+          - name: NUXEO_CUSTOM_PARAM
+            value: |
+              nuxeo.mongodb.server=mongodb://${APPLICATION_NAME}-mongo-0.${APPLICATION_NAME}-mongo:27017
+              nuxeo.mongodb.dbname=${APPLICATION_NAME}-dev
+              elasticsearch.client=RestClient
+              elasticsearch.httpReadOnly.baseUrl=http://${APPLICATION_NAME}-elasticsearch:9200
+              elasticsearch.indexName=${APPLICATION_NAME}-dev
+              mail.transport.host=aws-smtp-relay.common.svc
```

Comment:
Use `aws-smtp-relay.common-infra.svc` for prod cluster.

Reply:
@dmetzler How to do so? I just did as in `nuxeo-ps-template.yaml`. Should it be changed?

Reply:
done, and also opened other pull requests for the other templates.